modules/core/arrow-annotations-processor/src/main/java/arrow/common/utils/ProcessorUtils.kt

```diff
@@ -47,7 +47,8 @@ interface ProcessorUtils : KotlinMetadataUtils {
   }
 
   fun getClassOrPackageDataWrapper(classElement: TypeElement): ClassOrPackageDataWrapper {
-    val metadata = classElement.kotlinMetadata ?: knownError("Arrow's annotations can only be used on Kotlin classes. Not valid for $classElement")
+    val metadata = classElement.kotlinMetadata
```

Comment:
what's the code style we're using for this repo?

Reply:
Not sure. I reformatted project with 2 spaces in my last commit and it moved this on two lines. Since I was unsure what the project style is for this case I left it so.